cmd/derper/cert.go

```diff
@@ -24,6 +24,7 @@ import (
 	"regexp"
 	"time"
 
+	upstreamacme "golang.org/x/crypto/acme"
```

Comment:
✍️ Style (non-blocking): Why the longer import alias? We are cool with `autocert` so why not plain `acme`?
```suggestion
	"golang.org/x/crypto/acme"
```